android/core/src/main/java/com/pega/vehicledamagemodeling/api/SelectionService.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 Flying Vehicle Monster team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.pega.vehicledamagemodeling.api;
+
+import com.badlogic.gdx.graphics.g3d.Material;
+import com.google.gson.JsonObject;
+
+public class SelectionService {
+    private SelectedPartsRepository selectedPartsRepository = new SelectedPartsRepository();
```

Comment:
This is violating the Dependency Inversion Principle. SelectedPartsRepository should be injected in the constructor.